mxfusion/components/factor.py

```diff
@@ -13,30 +13,39 @@
 # ==============================================================================
 
 
-# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Accidentally removed this copyright :)

Reply:
It is not an accident. This file somehow has two copyright notices. :)